app/grandchallenge/components/models.py

```diff
@@ -662,30 +662,42 @@ class OverlaySegmentsMixin(models.Model):
 
     @property
     def voxel_values(self):
-        allowed_values = {x["voxel_value"] for x in self.overlay_segments}
+        return {x["voxel_value"] for x in self.overlay_segments}
 
+    @property
+    def allowed_values(self):
```

Comment:
Do we need to split `voxel_values` and `allowed_values`? It's confusing having two now.